src/components/table/table.tsx

```diff
@@ -771,7 +771,7 @@ export class Table {
             >
                 <limel-checkbox
                     onChange={this.selectAllOnChange}
-                    disabled={!this.totalRows}
+                    disabled={!this.totalRows && !this.data.length}
```

Comment:
Hmm… I wonder why we didn't just look at `this.data.length` in the first place? 🤔

Reply:
This was implemented in #1632, fixing Lundalogik/crm-feature#2673. The intention is clearly to disable the checkbox is there are no rows currently being shown.

```suggestion
                    disabled={!this.data.length}
```

Reply:
Done!